plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererInvocationStatus.kt

```diff
@@ -69,9 +59,9 @@ class CodeWhispererInvocationStatus {
         return timeCanShowCodeWhisperer.isBefore(Instant.now())
     }
 
-    fun isPopupActive(): Boolean = isPopupActive
+    fun isDisplaySessionActive(): Boolean = isPopupActive
```

Comment:
is it different from "Session.isDisposed()"?

Reply:
asking because i feel they kinda serve similar purpose

Reply:
It's still different because it's a valid case that a session is active but displaySession is not active. 
We start the session as soon as a trigger happens to track the current ongoing requests, but all suggestions can be discarded so a displaySession can never be active but we still need to dispose the current valid session.

Reply:
thanks for clarifying it